resources/puppetlabs/lein-ezbake/template/foss/ext/redhat/init.erb

```diff
@@ -131,6 +131,13 @@ rh_status_q() {
     rh_status >/dev/null 2>&1
 }
 
+reload() {
+    echo -n $"Reloading $prog: "
```

Comment:
Not that I have any issue with it, but I'm curious why there is an echo here when there is not one in most of the other scenarios.